recipes/rdkit/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "rdkit" %}
+{% set version = "Release_2017_09" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/rdkit/rdkit/archive/{{ version }}.tar.gz
+  sha256: ab83bb4d3c5986d22da0ce28d352e5f911830b63f18fee85537cebfa3df28e82
+  patches:
+    - rdpaths.patch
+    - rdconfig.patch  # [win]
```

Comment:
Have these been sent upstream? What do they do?

Reply:
great question, I don't really know. This was adapted from a [previous conda recipe](https://github.com/rdkit/conda-rdkit) written by @greglandrum, who would have a better idea.

Reply:
The paths patch at least seems to be present in CMakeLists.txt in the main repo, so I'm going to try building without them.